README.md

```diff
@@ -1,3 +1,91 @@
 # opentelem-python
 
-Open Telemetry Codecov Python Prototype
+_Note: This package is part of the [Runtime Insights Early Access Program](https://about.codecov.io/product/feature/runtime-insights/)._
+
+## Purpose
+
+This package allows Python projects to leverage Codecov's [Runtime Insights](https://about.codecov.io/product/feature/runtime-insights/) feature.
+
+## Requirements and Pre-requisites
+
+1. A repository that is active on [Codecov](https://codecov.io)
+2. A profiling token for that repository, obtainable from Codecov.
+3. Python version >=3.6
+
+## Installation
+
+First, install the package:
+
+```
+pip install codecovopentelem
+```
+
+Second, include the following snippet in your application's startup / bootstrapping process:
```

Comment:
I understand that this is general the right thing. But I wonder if clarifying "Wherever you would put your opentelemetry startup code" works. I also don't feel comfortable to have the default recommendation do everything at import time (i.e outside of a startup function that is called by the right hook)

Reply:
This is arguably the part where I have the weakest knowledge regarding this package. I'm open to stating whatever you feel comfortable saying, just let me know in explicit terms what that is. In other words, give me a couple sentences and I'll put them here 😄 .